### ISSUE TITLE
train MAE on my won dataset (4.7w images), loss decreased from 1.8 to 0.6, the reconstruction result is terrible?

### ISSUE THREAD

**Issue by Mogul5306** [facebookresearch/mae#95]
batch size=32
lr=1.5e-4
weight_decay=0.05
adamW beta1=0.9, beta2=0.95
epochs=400
warm epoch=40

**Comment by onlyonewater** [comment]
so, Does mae not work well on small datasets?

**Comment by Mogul5306** [comment]
Yeah，the loss drops to about 0.54, and the reconstructed images looks like colorful noise.

**Comment by onlyonewater** [comment]
all right, Looks like mae needs some tricks on small datasets.

**Comment by onlyonewater** [comment]
hi, @Mogul5306 ,I think you can use the pretrained mae, and fine-tuning on your own dataset, it may be helpful.

**Comment by Mogul5306** [comment]
ok，i’ll try it. But there is a problem: my dataset is very different from ImageNet. it is an particular industrial dataset. If it works well, how can I explain it?

**Comment by onlyonewater** [comment]
if your own dataset is very different from imagenet (have a huge gap), I think you may not get good results by using the pre-trained + fine-tuning method directly. you can use some transfer learning methods. but I do not know what are the methods. or you can use some data augmentation methods to increase your own dataset size, or learn how vit (vision image transformer) was successfully trained on a small dataset (https://openreview.net/pdf?id=SCN8UaetXx, https://arxiv.org/pdf/2112.13492.pdf).

**Comment by jhanmtl** [comment]
I had somewhat similar issues and running without the --norm_pixel_loss flag seems to yield better results. Can you try that?

**Comment by xiangtaowong** [comment]
hello,
any tutorial for custom dataset? 
thank u!

**Comment by FabianSchuetze** [comment]
I have similar problems that the results when training from afresh on a small dataset look like colorfull noise with the loss hovering around 0.5-0.6. I used the official learning rate but did adjust the accumulation period accordingly. So far, I have trained with setting norm_pixel_loss to False (as are the defaults) but I use timm's small VIT. What model did you try @jhanmtl and for how long did you train? Did you adjust the learning rate somehow?

**Comment by jerry-dream-fu** [comment]
i use the pretrained mae, and fine-tuning on my own dataset. the results in downstream task seems more bad than direct using pretrained mae in downstream task!!!

**Comment by jerry-dream-fu** [comment]
> I have similar problems that the results when training from afresh on a small dataset look like colorfull noise with the loss hovering around 0.5-0.6. I used the official learning rate but did adjust the accumulation period accordingly. So far, I have trained with setting norm_pixel_loss to False (as are the defaults) but I use timm's small VIT. What model did you try @jhanmtl and for how long did you train? Did you adjust the learning rate somehow?

If setting norm_pixel_loss to False, there should be no colorfull noise in my code

**Comment by LeroyChou** [comment]
such a great advice. I've encountered the same problem, and by setting norm_pixel_loss to False I remove the colored noise. But I don't really understand why. Does anybody have some ideas?

**Comment by Marquess98** [comment]
> batch size=32 lr=1.5e-4 weight_decay=0.05 adamW beta1=0.9, beta2=0.95 epochs=400 warm epoch=40

Hi，could you please help me in how to get the visualize checkpoint fine-tuning in custom dataset? cause I fould the checkpoint in output_dir file only contains encoder. thank you so much !

**Comment by bibhabasumohapatra** [comment]
@Mogul5306  Bro I think Like the Loss always decreases because the default mask_ratio and loss implementation, I see that the implementation they find loss between predicted image and masked input and not the original image . . . how would the model learn if the GT is not the Image itself.
I'm got here because faced the same problem that decreased loss but no reconstruction

**Comment by yyy1998-i** [comment]
> 我使用预先训练的 MAE，并在我自己的数据集上进行微调。在 Downstream Task 中，结果似乎比在 Downstream Task 中使用预训练的 MAE 更糟糕!!
请问您解决这个问题了吗，我也遇到了相似的问题
